C/plugins/storage/sqlite/common/connection_manager.cpp

```diff
@@ -125,6 +125,59 @@ Connection *conn = 0;
 	return conn;
 }
 
+bool ConnectionManager::attachNewDb(std::string &path, std::string &alias)
```

Comment:
Could you add a function header comment here please.

Reply:
done